tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -520,6 +520,49 @@ def test_error_handled_condition(self):
             assert [None] == response.data["data"][0]["error.handled"]
             assert [1] == response.data["data"][1]["error.handled"]
 
+    def test_error_unhandled_condition(self):
+        self.login_as(user=self.user)
+        project = self.create_project()
+        prototype = load_data("android-ndk")
+        events = (
+            ("a" * 32, "not handled", False),
+            ("b" * 32, "was handled", True),
+            ("c" * 32, "undefined", None),
+        )
+        for event in events:
+            prototype["event_id"] = event[0]
+            prototype["message"] = event[1]
+            prototype["exception"]["values"][0]["value"] = event[1]
+            prototype["exception"]["values"][0]["mechanism"]["handled"] = event[2]
+            prototype["timestamp"] = self.two_min_ago
+            self.store_event(data=prototype, project_id=project.id)
+
+        with self.feature("organizations:discover-basic"):
+            query = {
+                "field": ["message", "error.unhandled", "error.handled"],
+                "query": "error.unhandled:true",
+                "orderby": "message",
+            }
+            response = self.do_request(query)
+            assert response.status_code == 200, response.data
+            assert 1 == len(response.data["data"])
+            assert [0] == response.data["data"][0]["error.handled"]
```

Comment:
Why is `error.handled` an array while `error.unhandled` an integer?

Reply:
`error.handled` is stored in the schema as Array(UInt8) because an error event can have multiple exceptions (chained exceptions) and each of those exceptions could be flagged as 'handled'. We're working on getting to a simpler model where an error event is either handled or unhandled, and this is one of those steps. For more context see https://www.notion.so/sentry/Crashes-in-SDKs-and-Product-fa249e31d6274b66b1bfff966e00b668

If this prototype of error.unhandled is well received we'll likely make changes to storage so that error.unhandled and error.handled come from the same boolean value.